routes/classesRouter.js

```diff
@@ -0,0 +1,70 @@
+const classesRouter = require("express").Router()
+const mySchool = require("./../School.js")
+const Student = require("./../Student.js")
+
+classesRouter.post("/", (req, res) => {
+  res.json(req.body)
+})
+
+classesRouter.post("/add", (req, res) => {
+  let classNames = req.body.name
+  if (mySchool.classes[classNames]) {
+    res.json({
+      error: "Please fill out all the information",
+      timestamp: "YYYY, MM/DD HH:MM:SS"
+    })
+  } else {
+    mySchool.addClass(req.body.name, req.body.teacher)
+    res.json({
+      class: {
+        name: `${req.body.name}`,
+        teacher: `${req.body.teacher}`,
+        students: []
+      },
+      message: "Created a new class",
+      timestamp: "YYYY, MM/DD HH:MM:SS"
+    })
+  }
+})
+
+classesRouter.get("/:className/students", (req, res) => {
```

Comment:
The callback functions line 30 and 40 on could be middleware that redirects the request to the specific filter